content/applications/hr/payroll/work_entries.rst

```diff
@@ -2,75 +2,86 @@
 Work entries
 ============
 
-The *Work Entries* dashboard, which can be found by going to :menuselection:`Payroll --> Work
-Entries --> Work Entries`, provides a visual overview of the individual time sheets of every
-employee, with each day split into a morning shift and an afternoon shift.
+Work entries are created automatically in the *Payroll* app, based on the employee's :ref:`salary
+structure type <payroll/structure-types>`, and from the *Planning*, *Attendances*, and *Time Off*
+applications.
+
+The *Work Entries* dashboard of the *Payroll* application provides a visual overview of the
+individual work entries for every employee.
+
+To open the dashboard, navigate to :menuselection:`Payroll app --> Work Entries --> Work Entries`.
+
+On the :guilabel:`Work Entry` dashboard, work entries appear in alphabetical order, based on the
+first name of the employees. The entire month is displayed, with the current day highlighted in pale
+yellow.
+
+If any entries have :ref:`conflicts <payroll/conflicts>` that need to be resolved, the dashboard
+defaults to filter only the `Conflicting` entries.
```

Comment:
:guilabel:\`Conflicting\` was used later in the doc, and I like using the :guilabel: when referring to the active filters. Can use literal formatting when typing in to apply a filter.
```suggestion
If any entries have :ref:`conflicts <payroll/conflicts>` that need to be resolved, the dashboard
defaults to filter only the :guilabel:`Conflicting` entries.
```